.github/workflows/wsl-helper-build.yaml

```diff
@@ -0,0 +1,45 @@
+# This workflow builds the WSL Helper
+name: 'GitHub Runner: WSL Helper'
+
+on:
+  push:
+    paths: [ src/go/wsl-helper/** ]
+  pull_request:
+    paths: [ src/go/wsl-helper/** ]
+  workflow_dispatch:
+
+permissions:
+  # This uses external actions, lock things down to read-only.
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
As you mentioned, we should add a Windows build too. (Probably by setting `GOOS` but still using Linux runners)